mmdet3d/core/evaluation/seg_eval.py

```diff
@@ -66,28 +66,38 @@ def get_acc_cls(hist):
     return np.nanmean(np.diag(hist) / hist.sum(axis=1))
 
 
-def seg_eval(gt_labels, seg_preds, label2cat, logger=None):
-    """Semantic Segmentation Evaluation.
+def seg_eval(gt_labels, seg_preds, label2cat, ignore_index, logger=None):
+    """Semantic Segmentation  Evaluation.
 
      Evaluate the result of the Semantic Segmentation.
 
      Args:
```

Comment:
It seems the indent of Args and the previous sentence is inconsistent? Should be aligned with """?

Reply:
Fix.